src/search/FindUtils.js

```diff
@@ -522,6 +522,28 @@ define(function (require, exports, module) {
         };
     }
 
+    /**
+     * Parses a string and replace the \r, \n and \t sequences of characters
+     * with a character based on the regex meaning of these sequences.
+     * \n => Line Feed
+     * \r => Carriage Return
+     * \t => Tab
+     */
+    function parseString(string) {
+        return string.replace(/\\(.)/g, function (match, ch) {
+            if (ch === "n") {
+                return "\n";
+            }
+            if (ch === "r") {
+                return "\r";
+            }
+            if (ch === "t") {
+                return "\t";
+            }
+            return ch;
+        });
+    }
```

Comment:
@ficristo This is an awesome improvement!

I think the name `parseString()` is too generic, though. Maybe `regexUnescapeString()` ?

Does this need to detect when the backslash itself escaped so that you can (for example) use "\\n" to insert "\n" (i.e. not a newline)? Either way, that would be a good unit test.